docs/en/observability/profiling-get-started.asciidoc

```diff
@@ -111,7 +111,7 @@ We still recommend upgrading as the latest version contains several improvements
 
 [discrete]
 [[profiling-install-host-agent]]
-== Install the host-agent
+== Install the host-agent in standalone mode

```

Comment:
I think it's probably a good idea to tell people that there are two options before this section so they don't start with standalone and realize they want to install with the agent. I tried to do this in suggestions, but wasn't sure I was getting the formatting right, so I'll give it a try here:

```
[discrete]
[[profiling-install-host-agent]]
== Install the host-agent

You have the following options when installing the host-agent:

- <<profiling-install-host-agent-standalone>>
- <<profiling-install-host-agent-elastic-agent>>

[discrete]
[[profiling-install-host-agent-standalone]]
=== Install the host-agent in standalone mode
```

Reply:
done